packages/client/tests/e2e/_utils/run.ts

```diff
@@ -51,45 +51,66 @@ async function main() {
   console.log('🎠 Preparing e2e tests')
   // we first get all the paths we are going to need to run e2e tests
   const tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), 'prisma-build'))
+
   const cliPkgPath = path.join(monorepoRoot, 'packages', 'cli')
   const wpPluginPkgPath = path.join(monorepoRoot, 'packages', 'nextjs-monorepo-workaround-plugin')
   const clientPkgPath = path.join(monorepoRoot, 'packages', 'client')
+  const enginesPkgPath = path.join(monorepoRoot, 'packages', 'engines')
+  const debugPkgPath = path.join(monorepoRoot, 'packages', 'debug')
+  const generatorHelperPkgPath = path.join(monorepoRoot, 'packages', 'generator-helper')
+
   const cliPkgJsonPath = path.join(cliPkgPath, 'package.json')
   const clientPkgJsonPath = path.join(clientPkgPath, 'package.json')
+  const generatorHelperPkgJsonPath = path.join(generatorHelperPkgPath, 'package.json')
+
   const cliPkgJson = require(cliPkgJsonPath)
   const clientPkgJson = require(clientPkgJsonPath)
+  const generatorHelperPkgJson = require(generatorHelperPkgJsonPath)
 
   // this process will need to modify some package.json, we save copies
   await $`cd ${tmpDir} && cp ${cliPkgJsonPath} cli.package.json`
   await $`cd ${tmpDir} && cp ${clientPkgJsonPath} client.package.json`
+  await $`cd ${tmpDir} && cp ${generatorHelperPkgJsonPath} generator-helper.package.json`
 
   // we provide a function that can revert modified package.json back
   const restoreOriginal = async () => {
     await $`cd ${tmpDir} && cp cli.package.json ${cliPkgJsonPath}`
     await $`cd ${tmpDir} && cp client.package.json ${clientPkgJsonPath}`
+    await $`cd ${tmpDir} && cp generator-helper.package.json ${generatorHelperPkgJsonPath}`
   }
 
   // if process is killed by hand, ensure that package.json is restored
   process.on('SIGINT', () => restoreOriginal().then(() => process.exit(0)))
 
-  // use bundleDependencies to directly include deps like @prisma/engines
```

Comment:
`bundleDependencies` didn't work with transitive dependencies of `@prisma/debug`, and those were missing after package installation.

I haven't tried recursively reading `package.json` files and adding transitive dependencies to `bundleDependencies` as well — perhaps that could have worked.

Instead, I replaced references to `@prisma/engines` and `@prisma/debug` with packaged tarballs. One downside of this approach that I can see is that since I used absolute paths, now they are only usable inside the docker container.

Reply:
Could you please open a discussion with Zoltan in the pnpm repo? (Otherwise I can do it if you have other priorities)

Reply:
forgot to post the link here, https://github.com/orgs/pnpm/discussions/6861

Reply:
I just made that everything is put in /tmp locally and in the docker container, so the difference is eliminated for macOS and Linux.